tanner/utils/sqlite_db_helper.py

```diff
@@ -49,6 +60,12 @@ def copy_db(self, src, dst, working_dir):
         return dst
 
     async def insert_dummy_data(self, table_name, data_tokens, cursor):
+        """
+        Inserts Dummy data in the current sqlite database
+        :param table_name (str): Table to inject dummy data
+        :param data_tokens (str): Tokens to generate dummy data, ex: 'I,L' to get ID and username data
```

Comment:
`ex` --> `eg`. Also, I'm not sure if I,L stands for ID and username. Please confirm it.

Reply:
Yes, it does - https://github.com/mushorg/tanner/blob/9fa7289896d194e0ca7b03faec3dfd72553271a8/tanner/utils/base_db_helper.py#L28